CONTRIBUTING.md

```diff
@@ -43,6 +44,30 @@ We can use help in a bunch of areas and any help is greatly appreciated!
     + [Maintainers team](#maintainers-team)
     + [Past Maintainers](#past-maintainers)
 
+## AI assistance notice
+
+> [!IMPORTANT]
+>
+> If you are using **any kind of AI assistance** to contribute to Biome,
+> it must be disclosed in the pull request.
+
+If you relied on AI assistance to make a pull request, you must disclose it in the
+pull request, together with the extent of the usage. For example, if you used
+AI to generate docs or tests, you must say it.
+An example disclosure:
+
+- > This PR was written primarily by Claude Code.
+- > I consulted ChatGPT to understand the codebase but the solution
+  > was fully authored manually by myself.
+
+Failure to disclose this is first and foremost rude to the human operators
+on the other end of the pull request, but it also makes it difficult to
+determine how much scrutiny to apply to the contribution.
```

Comment:
```suggestion
Providing this information helps reviewers understand the context of the
pull request and apply the right level of scrutiny, ensuring a smoother
and more efficient review process.
```
The original paragraph felt a bit accusatory to me, how do you like this wording?